client-js/examples/install.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  CEP78Client,
+  NFTOwnershipMode,
+  NFTKind,
+  NFTMetadataKind,
+  NFTIdentifierMode,
+  MetadataMutability,
+  OwnerReverseLookupMode,
+  MintingMode,
```

Comment:
I think `BurnMode` is missing here, please double check the modality parity with what is on `dev`

Reply:
oh I made it optional, just checking all of the modalities

Reply:
https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/98/files#diff-f014c08f47e43401e428e6acbb68846c9bdaac6e9520385776107e718dc7dddc

here are all modalities